docs/hub/spaces-overview.md

```diff
@@ -97,7 +97,10 @@ Paused time is not billed.
 
 In some cases, you might be interested in having programmatic access to the Space author or repository name. This feature is particularly useful when you expect users to duplicate your Space. To help with this, Spaces exposes different environment variables at runtime. Given a Space [`osanseviero/i-like-flan`](https://huggingface.co/spaces/osanseviero/i-like-flan):
 
+* `CPU_CORES`: 4
+* `MEMORY`: 15Gi 
```

Comment:
they're not namespaced by `SPACE_`?

Reply:
iguess it makes sense, they're not part of the Space conceptually, but of the runtime 🤷